cluster/manifests/skipper/hpa.yaml

```diff
@@ -25,6 +25,22 @@ spec:
       target:
         type: Utilization
         averageUtilization: {{ .ConfigItems.skipper_ingress_target_average_utilization_memory }}
+{{ if eq .ConfigItems.skipper_time_based_scaling_enabled "true" }}
+  - type: External
+    external:
+      target:
+        averageValue: {{ .ConfigItems.skipper_time_based_scaling_target }}
+        type: AverageValue
+      metric:
+        name: zmon-check
+        selector:
+          matchLabels:
+            type: "zmon"
+            check-id: {{ .ConfigItems.skipper_time_based_scaling_check_id }}
```

Comment:
This should be quoted I think, or does it work even if it's an int?

Reply:
good point. I pushed a change also removing the `skipper_time_based_scaling_enabled` as we talked.